### ISSUE TITLE
Beispiel für Aufzählungen korrigieren

### ISSUE THREAD

**Issue by beistehen** [geostandards-ch/doc_refhb24#8]
[Kapitel 2.8.2](https://geostandards-ch.github.io/doc_refhb24/#_aufz%C3%A4hlungen)

![grafik](https://github.com/user-attachments/assets/c1c932d2-4517-4e57-b8b5-3bdc29fed34f)

*Wo sind die Werte für **rot** und **gruen** im Wertebereich FarbePlusPlus?*

**Comment by zigertiger** [comment]
Ich verstehe diese RefMan-Passage so: `FarbePlusPlus` enthält alle Werte von `FarbePlus`, aber neu `#blau.hellblau` und `#blau.dunkelblau` **statt** `#blau` (im Gegensatz zu `FarbePlus`, wo `blau` ein gültiger Wert ist. Somit wäre das RefMan korrekt (?)

**Comment by beistehen** [comment]
> Ich verstehe diese RefMan-Passage so: `FarbePlusPlus` enthält alle Werte von `FarbePlus`, aber neu `#blau.hellblau` und `#blau.dunkelblau` statt #blau

Genau, das sehe ich auch so. Deshalb ist ja meine Kritik an diesem Beispiel, dass die Werte für `rot` und `gruen` aus FarbePlus nicht in der Liste der zulässigen Werte aufgeführt sind.

In den Beispielen sollten die "_mittels Konstanten beschriebenen - zulässigen Werte_" aufgeführt sein.

Für `FarbePlusPlus` wären das m. E. folgende Konstanten:

* `#blau.hellblau`
* `#blau.dunkelblau`
* `#rot.dunkelrot`
* `#rot.orange`
* `#rot.karmin`
* `#gruen.hellguen`
* `#gruen.dunkelgruen`

Aufgeführt sind aber nur `#blau.hellblau` und `#blau.dunkelblau` (und der Text "_statt `#blau`_" gehört ja auch nicht in die Liste der Konstanten ...)
